<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnVision_baseUI</class>
 <widget class="QMainWindow" name="EnVision_baseUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>1206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EnVision</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_Maintenance">
       <layout class="QGridLayout" name="gridLayout_9"/>
      </widget>
      <widget class="QWidget" name="page_Data">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QMdiArea" name="mdiArea">
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="background">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="viewMode">
           <enum>QMdiArea::TabbedView</enum>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="tabsMovable">
           <bool>true</bool>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <widget class="QWidget" name="Plotter">
           <property name="windowTitle">
            <string>Plotter</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3"/>
          </widget>
          <widget class="QWidget" name="Data">
           <property name="windowTitle">
            <string>Data</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4"/>
          </widget>
          <widget class="QWidget" name="Cube">
           <property name="windowTitle">
            <string>Cube</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8"/>
          </widget>
          <widget class="QWidget" name="Image">
           <property name="windowTitle">
            <string>Image</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_11"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1347</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCascade"/>
    <addaction name="actionTile"/>
    <addaction name="actionOverview"/>
   </widget>
   <widget class="QMenu" name="menuWork_on">
    <property name="title">
     <string>Work on</string>
    </property>
    <addaction name="actionColorma"/>
    <addaction name="actionRotate"/>
    <addaction name="actionZoom"/>
    <addaction name="actionPan"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_state"/>
    <addaction name="actionReload_state"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImport"/>
   <addaction name="menuExport"/>
   <addaction name="menuView"/>
   <addaction name="menuWork_on"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>menuBar_01</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar_Left</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSwitchTileOrCascade"/>
   <addaction name="separator"/>
   <addaction name="actionToggleDockWidget03"/>
   <addaction name="actionToggleDockWidget02"/>
   <addaction name="actionToggleDockWidget01"/>
   <addaction name="separator"/>
   <addaction name="actionToggleTableWidget"/>
   <addaction name="actionTogglePlotWidget"/>
   <addaction name="actionToggleImageWidget"/>
   <addaction name="actionToggleCubeWidget"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_01">
   <property name="windowTitle">
    <string>DockWidget01</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_5"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_02">
   <property name="windowTitle">
    <string>DockWidget02</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_10"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_03">
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>390</height>
    </size>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>File Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidget_filedialog">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Terminal</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="documentTitle">
             <string>THIS IS A TEST</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
            <property name="placeholderText">
             <string>LOL</string>
            </property>
           </widget>
           <widget class="QListWidget" name="listWidget_term">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <property name="uniformItemSizes">
             <bool>false</bool>
            </property>
            <property name="selectionRectVisible">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>gwegwgwe</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionOverview">
   <property name="text">
    <string>Overview</string>
   </property>
  </action>
  <action name="actionColorma">
   <property name="text">
    <string>Overlay</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_state">
   <property name="text">
    <string>Save state</string>
   </property>
  </action>
  <action name="actionSwitchTileOrCascade">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tab Layout</string>
   </property>
   <property name="toolTip">
    <string>CTRL+T: Change Tab Layout</string>
   </property>
  </action>
  <action name="actionToggleDockWidget03">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File Explorer</string>
   </property>
   <property name="toolTip">
    <string>Hide/Show File Dialog</string>
   </property>
  </action>
  <action name="actionToggleDockWidget02">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DockWidget02</string>
   </property>
   <property name="toolTip">
    <string>Hide/Show Widget02</string>
   </property>
  </action>
  <action name="actionToggleDockWidget01">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DockWidget01</string>
   </property>
   <property name="toolTip">
    <string>Hide/Show Widget01</string>
   </property>
  </action>
  <action name="actionToggleImageWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Image</string>
   </property>
   <property name="toolTip">
    <string>Hide/Show Image</string>
   </property>
  </action>
  <action name="actionToggleTableWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Data</string>
   </property>
   <property name="toolTip">
    <string>Hide/Show Table</string>
   </property>
  </action>
  <action name="actionTogglePlotWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Plotter</string>
   </property>
   <property name="toolTip">
    <string>Hide/Show Plot</string>
   </property>
  </action>
  <action name="actionReload_state">
   <property name="text">
    <string>Reload state</string>
   </property>
  </action>
  <action name="actionToggleCubeWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Cube</string>
   </property>
   <property name="toolTip">
    <string>Hide/Show Cube</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/envision.qrc"/>
 </resources>
 <connections/>
</ui>
